### ISSUE TITLE
Players can't see/roll downtime activities

### ISSUE THREAD

**Issue by lulunacy** [Ethck/Ethck-s-Downtime-Tracking#88]
I just installed this mod and created some downtime activities for my players. I can see the activities when I am logged in as GM but my players cannot. Is this working as intended?

Attempts to correct: 
I deactivated every other module, no change. 
I tried on firefox and chrome, no change. 
Created new character, no change. 

![image](https://github.com/Ethck/Ethck-s-Downtime-Tracking/assets/113865523/27d285e9-6e07-4f84-9d28-7fd91c528e3b)
![image](https://github.com/Ethck/Ethck-s-Downtime-Tracking/assets/113865523/96098a61-748e-4f59-a23d-7c1722026887)

**Comment by Ethck** [comment]
You will need to create downtime activities as world downtime activities using that lower "+ Add World Downtime" option.

**Comment by lulunacy** [comment]
<blockquote>Downtime activities come in two different forms: local and global. Local downtimes are on a per actor basis and will not be shown in other actor's downtime tabs. Global downtimes will appear in every actor's downtime tabs. Both types of downtimes can be configured directly in the actor's character sheet. </blockquote>

So individual downtimes on a per actor basis is no longer a thing? Because as GM I can see the downtime activities on a character sheet but logging in as that character and accessing the character sheet that player owns under their log in, nothing is seen. 
DM View
![image](https://github.com/Ethck/Ethck-s-Downtime-Tracking/assets/113865523/f7defba8-ae38-46a3-836e-ea4bce7a355c)
Player View
![image](https://github.com/Ethck/Ethck-s-Downtime-Tracking/assets/113865523/04c375de-af09-4855-8b33-373ae70b4db0)

**Comment by Ethck** [comment]
No, that's absolutely still a feature. This is on the same actor? Try disabling the Crash's tracking and training compatibility feature in module settings.

**Comment by lulunacy** [comment]
Yes, same actor. Here is a screenshot of what I see as GM with every module except for Downtime active
![image](https://github.com/Ethck/Ethck-s-Downtime-Tracking/assets/113865523/98ec6aaf-a4c4-4949-902b-62cbb726967b)
And here is what the player sees on the same actor with only Downtime active 
![image](https://github.com/Ethck/Ethck-s-Downtime-Tracking/assets/113865523/d1135d4e-f14a-444d-bd84-141324e6dd59)

**Comment by Ethck** [comment]
Can you list for me your foundry version, d&d5e version, and the version of the module you have? Looks like I've got some bugs to go hunt.

**Comment by lulunacy** [comment]
Sure thing! 
Foundry: Version 11 Stable - Build 308
DnD5e: 2.3.1
Ethck's 5e Downtime Tracking: Version 0.7.2

**Comment by Ethck** [comment]
Okay, this should be fixed now. In v11 there was some changes in the backend that resulted in the `hidden` checkbox to be nonexistent when the data was saved to the actor. Because of this all activities were treated as hidden from players (but not from GMs!).
